components/prize_pool/commons/prize_pool.lua

```diff
@@ -342,7 +342,22 @@ PrizePool.additionalData = {
 				return tonumber(score)
 			end
 
-			local scores = Table.mapValues(Table.mapValues(mw.text.split(input, '-'), mw.text.trim), forceValidScore)
+			-- split the lastvsscore entry by '-', but allow negative scores
+			local rawScores = mw.text.split(input, '-')
+			local scores = {}
+			for index, rawScore in ipairs(rawScores) do
+				if String.isNotEmpty(rawScore) then
+					rawScore = mw.text.trim(rawScore)
+					table.insert(scores, rawScore)
+				elseif rawScores[index + 1] then
+					rawScores[index + 1] = mw.text.trim(rawScores[index + 1])
+					rawScores[index + 1] = '-' .. rawScores[index + 1]
+				else
+					error('Invalid entry "|lastvsscore=' .. input .. '"')
```

Comment:
Let's return empty instead of erroring. I know some wikis does `|lastvsscore=3-` for the first place (when bo5) before the match is played, as they know the winner will have a score of 3.